Dockerfile

```diff
@@ -18,15 +17,11 @@ RUN apt-get update && \
     libssl-dev \
     ca-certificates \
     miredo \
-    tor && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/* \
-    /tmp/* \
-    /var/tmp/*
```

Comment:
Why remove the `apt-get clean` and `rm -rf` statements?

Reply:
They are superfluous